ovs-dpdk.md

```diff
@@ -0,0 +1,227 @@
+# oVirt now supports OVS-DPDK
+
+
+[DPDK](http://dpdk.org/) (Data Plane Development Kit) provides high-performance packet processing libraries and user space drivers.
+Open vSwitch supports DPDK since v2.2 and starting with version 2.4, a DPDK-optimized vHost path is enabled in OVS.
+ 
+According to [Intel ONP performance tests](https://download.01.org/packet-processing/ONPS2.1/Intel_ONP_Release_2.1_Performance_Test_Report_Rev1.0.pdf), using OVS with DPDK can provide a huge performance enhancement, increasing network packets throughput and reduced latency.
+ 
+OVS-DPDK has been added to oVirt as an experimental feature.
+Please note that OVS-DPDK cannot access any features located within the linux kernel, for example: linux bridge,  tun/tap devices, iptables etc.
+
+## Requirements for DPDK:
+
+For best performance use an Intel Xeon class server system such as Ivy Bridge, Haswell or newer.
+Ensure that each memory channel has at least one memory DIMM inserted, and that memory size should be at least 4GB.
+ 
+The network card must be on the supported-vendor matrix, located here: <http://dpdk.org/doc/nics>
+
+
+## Configure OVS and DPDK on the Host
+
+* Make sure VT-d is enabled in BIOS
+* When adding a new host via oVirt GUI, add the following parameters to Kernel Command line:  “iommu=pt intel_iommu=on default_hugepagesz=1GB hugepagesz=1G hugepages=N”. 
+
+
+### Install dpdk packages from fast data path repo:
```

Comment:
I dropped any mention of yum repo since these packages exist in Fedora, Centos and RHEL official repos